server/src/main/java/org/elasticsearch/index/mapper/DocumentParser.java

```diff
@@ -414,6 +413,12 @@ static void parseObject(final ParseContext context, ObjectMapper mapper, String
         }
     }
 
+    static int getPositionForDynamicField(ParseContext context) {
+        // this does not return the exact column positions but column orderings
+        // which will be used for the actual column position calculations by ColumnPositionResolver.resolve()
+        return -1 - context.getDynamicMappers().size();
+    }
```

Comment:
```suggestion
    static int getPositionEstimate(ParseContext context) {
        // this does not return the exact column positions but column orderings
        // which will be used for the actual column position calculations by ColumnPositionResolver.resolve()
        return -1 - context.getDynamicMappers().size();
    }
```

The inline comment could probably also be changed to a javadoc? It's explaining behavior relevant for the caller, not clarifying an internal implementation detail.

Reply:
Thanks, agreed.